app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/auth_key_pair.rb

```diff
@@ -1,13 +1,15 @@
 class ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::AuthKeyPair < ManageIQ::Providers::CloudManager::AuthKeyPair
+  require "sshkey"
   supports :create
   supports :delete
 
   def self.raw_create_key_pair(ext_management_system, create_options)
     ext_management_system.with_provider_connection(:service => "PCloudTenantsSSHKeysApi") do |api|
       tenant_id = ext_management_system.pcloud_tenant_id(api.api_client)
       ssh_key   = IbmCloudPower::SSHKey.new(
-        :name    => create_options['name'],
-        :ssh_key => create_options['public_key']
+        :name        => create_options['name'],
+        :ssh_key     => create_options['public_key'],
+        :fingerprint => SSHKey.sha1_fingerprint(create_options['public_key'])
       )
 
       api.pcloud_tenants_sshkeys_post(tenant_id, ssh_key)
```

Comment:
@Kuldip-Nanda part of https://github.com/ManageIQ/manageiq/pull/21759 is to return a hash with the appropriate attributes needed to create an auth_key_pair record.

See https://github.com/ManageIQ/manageiq-providers-openstack/pull/784/files#diff-0e8e6c9331d12cabc425f7bd0135e291136e172135c5f5bc8a3c96e9b9f919aeR12 for an example